### ISSUE TITLE
Release tag failing

### ISSUE THREAD

**Issue by danirabbit** [elementary/actions#65]
## What Happened

It looks like tagging releases is broken

## Expected Behavior

We should get a release tag on GitHub

### Logs

See https://github.com/elementary/gala/runs/3305400983?check_suite_focus=true and in particular:

```
  "message": "Must specify access token via Authorization header. https://developer.github.com/changes/2020-02-10-deprecating-auth-through-query-param",
87
  "documentation_url": "https://docs.github.com/v3/#oauth2-token-sent-in-a-header"
```

**Comment by kgrubb** [comment]
I believe this might be related to https://github.com/elementary/actions/pull/31.

**Comment by kgrubb** [comment]
This ended up being resolved via #69 :tada:
